app/views/moves/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Editing move</h1>
```

Comment:
- [OPTIONA] I would recommend removing unused files from the scaffold, this way your code would be cleaner and easier to maintain.

Reply:
Done! 👍🏽  thanks @smunozmo  for the review